F/faust/build_tarballs.jl

```diff
@@ -176,7 +186,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("LLVM_jll", v"11.0.1"),
+    Dependency("LLVM_jll", v"12.0.1"),
```

Comment:
I think this should be a `BuildDependency`: julia already comes with its libllvm, not need to require it
```suggestion
    BuildDependency("LLVM_jll", v"12.0.1"),
```